<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertiesSettings</class>
 <widget class="QWidget" name="PropertiesSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>433</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel#titleLabel {
	padding-width-left:5px;
	padding-width-right:5px;
	border: 1px solid white;
	border-radius: 10px;
	min-width: 200px;
	min-height: 40px;
}

QLabel#statusLabel {
	background-color: white;
}

 QComboBox {
	selection-background-color: transparent;
	selection-color: black;
}

 QGroupBox {
     border: 1px solid gray;
     border-radius: 5px;
 }
QGroupBox::title {
     subcontrol-origin: margin;
     subcontrol-position: top left; /* position at the top center */
 }

QFrame#frame_1,#frame_2  {
	border: 2px solid rgb(125, 125, 125, 50%);
	border-radius: 3px;
	background-color:rgb(100, 200, 200, 50%);
}

QGroupBox QPushButton {
	min-width: 55px;
	min-height: 55px;
	max-width: 55px;
	max-height: 55px;
	border-image: url(:/Hac/round-button-small.png);
}

QPushButton#okButton, #cancelButton {
	border-image: url(:/Hac/background-rect-button.png);
	min-height: 40px;
	min-width: 80px;
}


QPushButton#upButton1, #upButton2 {
	image: url(:/Hac/up-arrow-black.png);
}
QPushButton#upButton1:pressed, #upButton2:pressed {
	image: url(:/Hac/up-arrow-red.png);
}
QPushButton#downButton1, #downButton2 {
	image: url(:/Hac/down-arrow-black.png);
}
QPushButton#downButton1:pressed, #downButton2:pressed {
	image: url(:/Hac/down-arrow-red.png);
}

</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="margin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Properties</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QGroupBox" name="acGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Air Conditioning</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="acRangeLabel">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Temperature Range:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="HacLineEdit" name="minLineEdit1">
          <property name="font">
           <font>
            <pointsize>25</pointsize>
           </font>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string> To</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="HacLineEdit" name="maxLineEdit1">
          <property name="font">
           <font>
            <pointsize>25</pointsize>
           </font>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QPushButton" name="downButton1">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="autoRepeat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="7">
         <widget class="QPushButton" name="upButton1">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="autoRepeat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="unitLabel">
          <property name="font">
           <font>
            <family>unifont</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>℃</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QLabel" name="unitLabel_3">
          <property name="font">
           <font>
            <family>unifont</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>℃</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QGroupBox" name="heatingGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Heating</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" rowspan="2">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="heatRangeLabel">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Temperature Range:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="ntcLabel">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>NTC:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <widget class="HacLineEdit" name="minLineEdit2">
          <property name="font">
           <font>
            <pointsize>25</pointsize>
           </font>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string> To</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="HacLineEdit" name="maxLineEdit2">
          <property name="font">
           <font>
            <pointsize>25</pointsize>
           </font>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QPushButton" name="downButton2">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="autoRepeat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="7">
         <widget class="QPushButton" name="upButton2">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="autoRepeat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="7">
         <widget class="QComboBox" name="maxComboBox_2">
          <property name="font">
           <font>
            <pointsize>25</pointsize>
           </font>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>Internal NTC</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>External NTC</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Internal/External NTC</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QLabel" name="unitLabel_4">
          <property name="font">
           <font>
            <family>unifont</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>℃</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="unitLabel_5">
          <property name="font">
           <font>
            <family>unifont</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>℃</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>87</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HacLineEdit</class>
   <extends>QLineEdit</extends>
   <header>haclineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/hac.qrc"/>
 </resources>
 <connections/>
</ui>
